src/ngx_http_lua_socket_tcp.c

```diff
@@ -782,6 +786,29 @@ ngx_http_lua_socket_tcp_connect(lua_State *L)
     return lua_yield(L, 0);
 }
 
+static int
+ngx_http_lua_socket_tcp_getsockfd(lua_State *L)
+{
+    ngx_connection_t                    *c;
+    ngx_http_lua_socket_tcp_upstream_t    *u;
```

Comment:
Style: variable identifiers must be vertically aligned up. See other similar places for examples.